packages/rainbowkit/src/components/RainbowKitProvider/provideRainbowKitChains.ts

```diff
@@ -144,7 +144,7 @@ export const provideRainbowKitChains = <Chain extends RainbowKitChain>(
     return {
       ...chain,
       name: defaultMetadata.name ?? chain.name, // favor colloquial names
-      iconUrl: chain.iconUrl ?? defaultMetadata.iconUrl,
-      iconBackground: chain.iconBackground ?? defaultMetadata.iconBackground,
+      iconUrl: defaultMetadata.iconUrl,
+      iconBackground: defaultMetadata.iconBackground,
     } as Chain;
```

Comment:
Since we will pass in `chains` from wagmi directly we won't need to have this `iconUrl` and `iconBackground`